test/framework/math/spatial_discretization/cg/sdm_test_01_continuous.cc

```diff
@@ -50,8 +50,7 @@ math_SDM_Test01_Continuous(const InputParameters& input_parameters)
   bool is_DG = false;
   {
     using namespace opensn;
-    if (sdm_type == "PWLC")
-      sdm_ptr = PieceWiseLinearContinuous::New(grid);
+    if (sdm_type == "PWLC") sdm_ptr = PieceWiseLinearContinuous::New(grid);
     else
       ChiInvalidArgument("Unsupported sdm_type \"" + sdm_type + "\"");
```

Comment:
@andrsd : should there sitll be a chiXXXXX at this stage?

Reply:
Yes. Issue #65 will take care of the names and probably more...